contrib/parseq-restli-client/src/main/java/RequestConfigKey.tokens

```diff
@@ -21,27 +21,29 @@ T__19=20
 T__20=21
```

Comment:
There are pros and cons of having generated code checked in. For example, it is sometimes useful to see in history how those files have changed. Also, it is more intuitive to be able to find source of every file on github vs having to generate some source files locally.
I would leave it as it is for now because it seems to be orthogonal to this PR and maybe create an issue to reconsider in the future if generated source files should be checked in?

Reply:
Sure. I didn't mean to address this in this pull request. Just would like to have consistent convention to follow.